<launch>    



    <node pkg="rosbot_task_manager" type="task_manager.py" name="task_manager" output="screen">
        <param name="yalm_path" type="string" value="$(find rosbot_task_manager)/params/command_params.yaml" />
    </node>
    
    <include file="$(find rosbot_task_manager)/launch/move_base_only.launch" />    
    <include file="$(find rosbot_task_manager)/launch/amcl_only.launch"/> 
    
    <node pkg="tf" type="static_transform_publisher" name="laser_broadcaster" args="0 0 0 3.14 0 0 base_link laser 100" />
    <node name="teleop_twist_keyboard" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" output="screen"/>


    <arg name="map_file" default="$(find rosbot_task_manager)/maps/map_task.yaml"/>       <!-- path of map file -->
    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" respawn="true" />
    
</launch>